src/pkrd/rng/gen6.rs

```diff
@@ -1,10 +1,52 @@
+use super::tinymt;
 #[derive(Clone, Copy, Debug, PartialEq, Default)]
 pub struct Gen6Rng {
+    init_seed: usize,
+    init_tinymt_state: [u32; 4],
+    last_mt_state_index: usize,
+    last_tinymt_state: [u32; 4],
     advances: u32,
+    index: u32,
```

Comment:
would tinymt_index be better? index is the term generally used to refer to your tinymt advance in tinyfinder/3dsrngtool

Reply:
renamed to mt_advances and tinymt_advances in the code, but remains advances and index in the display

Reply:
Normally I would completely agree, since there are lots of benefits to being consistent with other tools.

Some context for this ask:
Part of Reader's goal is to try and move the RNG community to use more correct terminology (e.g. `frame` to `advance` and `current seed` to `state`).  A lot of the terminology we use has been reused from past gens for consistency, but is incorrect for today's games.  Since the rush of new people from Gen 8, I realized this adds a barrier for new RNGers and it also makes it difficult for people to fully understand the underlying technology.

Consistency is usually a great reason to do something, however I think consistently incorrect is a great reason to change.

Reply:
with index specifically as far as im aware it is a new term to refer to tinymt advances, though its nice to be consistent. so moving forward, should ``Index`` on the display be changed to something like ``TinyMT Advances`` and ``Curr seed`` be chanced to ``Curr state``?

Reply:
Yep, that's correct!